### ISSUE TITLE
Suggest a Repository name on the "Create Repository" page.

### ISSUE THREAD

**Issue by BlenderDefender** [go-gitea/gitea#21132]
### Feature Description

GitHub has had this feature for quite a while now and i find it really useful, especially for creating test repositorys. Basically, when creating a new repository, GH suggests a unique repo name that consists of three random words, and when clicking on that suggestion, it fills in automatically. That would be nice to have for Gitea.

### Screenshots

Mock up:
![grafik](https://user-images.githubusercontent.com/61777131/189477314-ff084aff-6d51-4e43-b4da-4abd40e63ec0.png)

**Comment by delvh** [comment]
Really? I find that to be rather annoying:
Instead of something useful, I always have to read through this irrelevant section that does nothing else but waste my time.
Especially, even if you have to create test repos (which already happens basically never or at most once), you still cannot use the suggested name: You will never remember that that repo is a test repo with no other purpose because there is no indication in its name.
Additionally, it doesn't take much longer to navigate to the text field and input `test` (or something similar).

**Comment by BlenderDefender** [comment]
Yes, really. I don't understand what you mean with "irrelevant section", this proposal would add **five words** to the already existing text. Also, labeling test repos with test1, test2, test3, ... isn't that helpful either, when it comes to finding your test repo from five months ago.
